desci-server/src/services/journals/JournalFormService.ts

```
@@ -770,6 +770,7 @@ async function submitFormResponse(
         },
       });
 
+      debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)